client/starwhale/core/model/view.py

```diff
@@ -33,6 +46,59 @@ def recover(self, force: bool = False) -> t.Tuple[bool, str]:
     def info(self, fullname: bool = False) -> None:
         self._print_info(self.model.info(), fullname=fullname)
 
+    def diff(self, compare_uri: URI, show_details: bool) -> None:
```

Comment:
add `@BaseTermView._only_standalone`  decorator for diff function?

Reply:
fixed